docs/source/release_notes/release-notes.rst

```diff
@@ -52,6 +52,10 @@ develop
            It also now returns information about how much data was swept.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2409>`__)
 
+    *   - |improved|
+        - Exposes another version of getRanges that uses a configurable concurrency level, defaulting to 8.
```

Comment:
Could you add this parameter to the config docs (if suitable docs are present), link to them here, and include the actual name of the config setting in this release note, please?

Reply:
Doesn't look like we have any docs for the methods on the KVSConfig interface, just the Cassandra/DbKvs-specific ones. Should I add a section to https://palantir.github.io/atlasdb/html/configuration/key_value_service_configs/index.html with the two getRanges ones?

Reply:
A short section like that would be great 👍